spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config/README.md

```diff
@@ -37,77 +40,126 @@ spring.cloud.azure.appconfiguration.fail-fast | Whether throw RuntimeException o
 
 ### Watch configuration change
+
 Watch feature allows the application to load the latest property value from configuration store automatically, without restarting the application.
 
 By default, the watch feature is disabled. It can be enabled with below configuration:
-```
+
+```properties
 spring.cloud.azure.appconfiguration.watch.enabled=true
 ```
 
 Change certain property key in the configuration store on Azure Portal, e.g., /application/config.message, log similar with below will be printed on the console.
-```
+
+```console
 INFO 17496 --- [TaskScheduler-1] o.s.c.e.event.RefreshEventListener       : Refresh keys changed: [config.message]
 ```
+
 The application now will be using the updated properties. By default, `@ConfigurationProperties` annotated beans will be automatically refreshed. Use `@RefreshScope` on beans which are required to be refreshed when properties are changed.
 By default, all the keys in a configuration store will be watched. To prevent configuration changes are picked up in the middle of an update of multiple keys, you are recommended to use the watched-key property to watch a specific key that signals the completion of your update so all configuration changes can be refreshed together.
-```
+
+```properties
 spring.cloud.azure.appconfiguration.stores[0].watched-key=[my-watched-key]
 ```
 
 ### Failfast
+
 Failfast feature decides whether throw RuntimeException or not when exception happens. By default, failfast is enabled, it can be disabled with below configuration:
-```
+
+```properties
 spring.cloud.azure.appconfiguration.fail-fast=false
 ```
 
-### Use Azure Managed Identity to load the connection string
-[Managed service identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) allows application to access [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) protected resource on [Azure](https://azure.microsoft.com/en-us/).
+### Use Azure Managed Identity/Service Principle to load the connection string
+
+[Managed service identity](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview) allows application to access [Azure Active Directory](https://azure.microsoft.com/services/active-directory/) protected resource on [Azure](https://azure.microsoft.com).
 
-In this library, managed service identity is used to retrieve the connection string of the configuration store, the connection string is not required if running the Spring Boot application on Azure with managed service identity enabled.
+In this library, [Azure Identity SDK][azure_identity_sdk] is used to access Azure App Configuration and optionally Azure Key Vault, for secrets. The connection string is not required and will be ignored if Managed Identity is being used.
 
 Follow below steps to enable managed service identity feature:
```

Comment:
Follow below steps to enable *accessing App Configuration with* managed ~service~ identities ~feature~: